code/game/gamemodes/marker/evac_points.dm

```diff
@@ -32,12 +39,37 @@
 		evac_points = evac_threshold
 		//No addtimer call here, so we'll stop gaining points now
 	else
+		check_admin_warnings()
 		pointgain_timer = addtimer(CALLBACK(src, .proc/charge_evac_points), 1 MINUTE, TIMER_STOPPABLE) //Shuttle was unable to be called. Try again recursively.
 	return FALSE
 
-
+//proc that handles messages to admins regarding close evac time
+/datum/game_mode/marker/proc/check_admin_warnings()
+	var/time_till_evac = (evac_threshold - evac_points) * minutes_per_point MINUTES	//for Nanako, this formula here needs checking
+	if(time_till_evac <= 10 MINUTES)
+		if(show_admin_warning_10)
+			show_admin_warning_10 = FALSE
+			message_admins("Approximate time until evacuation is unlocked is less than 10 minutes.")
+		else
+			if(time_till_evac <= 5 MINUTES && show_admin_warning_5)
+				show_admin_warning_5 = FALSE
+				message_admins("Approximate time until evacuation is unlocked is less than 5 minutes.")
+
+/datum/game_mode/marker/proc/get_time_until_evac()
```

Comment:
Every usage of this function just multiplies it by MINUTES (which converts it to deciseconds) so you should do that here too instead of at the caller.

All time in byond is handled in deciseconds, best to stay consistent

Reply:
Added MINUTES to this proc, removed from places where it's called.